File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/AppVersionVO.kt
package com.sdkwork.app

data class AppVersionVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val currentVersion: String? = null,
    val latestVersion: String? = null,
    val updateRequired: Boolean? = null,
    val forceUpdate: Boolean? = null,
    val updateDescription: String? = null,
    val updateUrl: String? = null,
    val updateSize: String? = null,
    val releaseDate: Int? = null
)

File: sdkwork-app-sdk-typescript/src/types/app-version-vo.ts
/** 应用版本响应 */
export interface AppVersionVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 当前版本 */
  currentVersion?: string;
  /** 最新版本 */
  latestVersion?: string;
  /** 是否需要更新 */
  updateRequired?: boolean;
  /** 是否强制更新 */
  forceUpdate?: boolean;
  /** 更新描述 */
  updateDescription?: string;
  /** 更新链接 */
  updateUrl?: string;
  /** 更新大小 */
  updateSize?: string;
  /** 发布日期 */
  releaseDate?: number;
}
